test/appium/tests/medium/test_chats_m.py

```diff
@@ -739,6 +739,53 @@ def prepare_devices(self):
         for i in range(1, 3):
             self.chats[i] = self.homes[i].get_chat(self.chat_name).click()
 
+    @marks.testrail_id(702343)
+    def test_group_chat_send_delete_image(self):
+
+        self.chats[0].just_fyi("Sending image to group chat")
+        self.chats[0].show_images_button.click()
+        self.chats[0].allow_button.click()
+        self.chats[0].first_image_from_gallery.click()
+        self.chats[0].send_message_button.click()
+
+        self.chats[0].just_fyi("Verify sent image is displayed for every group member")
+        for i in range(1, 3):
+            if not self.chats[i].image_message_in_chat.is_element_displayed(20):
```

Comment:
increase time a bit; I think up to 1 min is fine

Reply:
thanx, done. Will re-run this group locally and see if it helped. The problem is that originally wait time was 10-15 seconds and it used to pass.